modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,77 @@ This release specifically highlights agility, performance, and manageability and
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
+[#release-552]
+== Release 5.5.2
+Couchbase Server 5.5.2, released in October 2018, is the second maintenance release in the 5.5.x series for Couchbase Server.
+This maintenance release includes fixes to the index and query services, XDCR, and installer components.
+
+[#fixed-issues-552]
+*Fixed Issues*
```

Comment:
Is there a reason we're not making this an H3?

Reply:
Good catch. Cleaning this up as the new site supports H3s now.